app/assets/javascripts/common/reports.js

```diff
@@ -1048,6 +1055,28 @@ Reports = function (withLtfu) {
       registrationsPeriodEndNodes.forEach(
         (node) => (node.innerHTML = periodInfo.bp_control_registration_date)
       );
+
+      const bs200To300breakdown = data.bs200To300BreakdownRates[period];
+      const bsOver300breakdown = data.bsOver300BreakdownRates[period];
+
+      bs200T0300RbsPPBSPercentNode.innerHTML = this.formatPercentage(
+          bs200To300breakdown ? bs200To300breakdown["random"] + bs200To300breakdown["post_prandial"] : 0
```

Comment:
The calculation can be separated into a separate method from the `innerHTML` update bit since doing this sort of calculation seems unusual for reports.js.